core/src/main/web/app/helpers/menupluginmanager.js

```diff
@@ -138,6 +141,9 @@
           });
         }
       }
+      if (bkUtils.isElectron && menuChanged){
+        bkUtils.Electron.updateMenus(menus);
```

Comment:
my favorite post
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html